redesign/cs-emf.html

```diff
@@ -0,0 +1,208 @@
+    <link href="https://fonts.googleapis.com/css?family=Istok+Web|Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="css/dwyl.css">
+    <link rel="shortcut icon" href="/img/favicon.ico">
+    <title>dwyl</title>
+  </head>
+  <body>
+    <nav class="w-100 dwyl-bg-dark-gray tc white relative fixed">
+      <input type="checkbox" id="burger" class="absolute top-2 right-1 dn">
+      <label for="burger" class="dn-ns pointer absolute top-1 right-1">
+        <i class="fa fa-bars white dib f2"></i>
+      </label>
+      <ul class="overflow-hidden menu db-ns w-100 list tc pl0 pt3 mv0 f3 fw3 f5-ns">
+        <li class="absolute top-1 static-ns ph4 mh2 fw3 di-ns pt1 pb3 pv3-ns">
+          <a href="/" class="white link">
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w1-plus"/>
+            dwyl
+          </a>
+        </li>
+        <li class="ph5 pt5 pb2 tl dn-ns"><a href="/" class="white link">Home</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/portfolio" class="white link">Portfolio</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/values" class="white link">Values</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/team" class="white link">Team</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/app" class="white link">App
+          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph5 di-ns pt2 pb3 tl pv0-ns"><a href="/#contact" class="white link">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="pt5 pb3 pb3-ns ph6-ns center tc shadow-3">
+      <img src="./img/common/ellen-macarthur-foundation-logo.png" alt="Ellen MacArthur Foundation logo" class="w5 pt4 mt4"/>
+      <h2 class="dwyl-light-gray f4">The Ellen MacArthur Foundation (EMF)</h2>
+      <h1 class="dwyl-dark-gray f3-plus w-90 tc center">Online Membership Platform</h1>
+    </section>
+
+    <section class="pt2 pb3 pv4-ns w-80 center">
+      <div class="w-40-ns dtc-ns v-top-ns">
+        <img src="./img/common/emf-phone-1.png" alt="A phone displaying an app on a challenges page">
+        <img src="./img/common/emf-phone-2.png" alt="A phone displaying an app on a page for selecting tags" class="dn db-ns">
+        <img src="./img/common/emf-phone-3.png" alt="A phone displaying an on a page to create a new challenge" class="dn db-ns">
+      </div>
+
+      <article class="w-50-ns dtc v-top istok-web lh-copy dwyl-dark-gray">
+        <h3 class="b mb2">Project: <span class="normal">CE100</span></h3>
+        <div class="mb4">
+          <a href="#" class="dib link mv2 pa2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">
+            <img src="./img/common/yellow-gray-heart.png" alt="dwyl heart logo" class="h1">
+            Live App
```

Comment:
I think we should do this, prospective clients want to see our previous work - any reasons why you think we shouldn't @SimonLab ?